funcbench/README.md

```diff
@@ -1,17 +1,22 @@
 # funcbench
 
-A tool used as a github action to run a `go test -bench` and compare changes from a PR against another branch or commit.
+A tool used as a GitHub action to run a `go test -bench` and compare changes from a PR against another branch or commit.
+
+> By default all benchmarks run without `-race` flag (#275).
 
 The benchmark can be triggered by creating a comment which specifies a branch to compare. The results are then posted back as a PR comment.
 The benchmark can be also trigger and used as CLI command, without GitHub hook.
 
 Comparison is done using [benchcmp](https://godoc.org/golang.org/x/tools/cmd/benchcmp).
-Arguments for the benchcmp are read from files created by previous action (for example [commentMonitor](/tools/commentMonitor/main.go)),
-which is responsible for the comment parsing.
+
+Arguments for the benchcmp are read from files created by previous action (for example [commentMonitor](/tools/commentMonitor/main.go)), which is responsible for the comment parsing.
```

Comment:
> Arguments for the benchcmp are read from files created by previous action

We're no longer doing this, we're getting the arguments from the `repository_dispatch` event.